adapter/src/components/styles/ErrorBoundary.style.js

```
@@ -8,11 +8,8 @@ const bgColor = '#F4F6F8',
 
 export default css`
     .mask {
-        position: absolute;
-        top: 0px;
-        left: 0px;
-        right: 0px;
-        bottom: 0px;
+        height: 100%;
+        width: 100%;
```

By default, the mask will fill up its container